src/fallback.ts

```diff
@@ -81,7 +81,7 @@ export class GrpcClient {
   authClient?: AuthClient;
   fallback: boolean | 'rest' | 'proto';
   grpcVersion: string;
-  private static protoCache = new Map<string, protobuf.Root>();
+  private static protoCache = new Map<string | Buffer, protobuf.Root>();
```

Comment:
Would it make sense to be more confident in the types of the internal data structure and ensure we always have a string key?

Reply:
Alternative way is to turn the `hash` value from Buffer to string. At [line](https://github.com/googleapis/gax-nodejs/blob/main/src/fallback.ts#L137)
`const hash = objectHash(json).toString()`;